src/main/webapp/WEB-INF/view/users.jsp

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
```

Comment:
see comment in UsersServlet.java

We can put the html of list of users (see spec) in the condition that the user is not logged in or url doesn't contain username.